lib/openstudio-standards/btap/structure.rb

```diff
@@ -512,14 +512,14 @@ def initialize(model = nil, cat = "commerce", lload = 30)
       # strictly for structural dimensioning/safety purposes. They are not (or
       # are very rarely) representative of actual day-to-day loads. Back of the
       # envelope calculations suggest reducing live load code requirements down
-      # to 1/8th of their initial values for internal mass purposes. These
+      # to ~1/12th of their initial values for internal mass purposes. These
```

Comment:
Approx. 1/12th of minimum NBC Part 4 live load requirements seems like a better fit. Something of a deep dive, including going over century-old reports like this [one](https://www.govinfo.gov/content/pkg/GOVPUB-C13-3fe62f845d95fcec4699463b39c5241d/pdf/GOVPUB-C13-3fe62f845d95fcec4699463b39c5241d.pdf) (under Herbert Hoover no less):

<img width="372" alt="lilveloads" src="https://github.com/user-attachments/assets/d4bb2a3f-32d2-4391-afe6-a787c2e5b489" />